package.json

```diff
@@ -11,14 +11,50 @@
     "url": "git://github.com/conveyal/otp.js.git"
   },
   "devDependencies": {
-    "component": "*",
-    "component-uglifyjs": "*",
+    "babel-plugin-syntax-dynamic-import": "^6.18.0",
+    "backbone": "^1.2.1",
+    "bootstrap": "^3.3.5",
+    "eonasdan-bootstrap-datetimepicker": "^3.1.3",
+    "css-loader": "^2.1.1",
+    "es5-shim": "^4.5.13",
+    "es6-shim": "^0.35.0",
+    "file-loader": "^3.0.1",
+    "handlebars": "^4.1.2",
+    "haversine": "^1.1.1",
+    "html-loader": "^0.5.5",
+    "inquirer": "^6.3.1",
+    "jquery": "^1.11.3",
```

Comment:
A number of these dev dependencies seem to actual dependencies rather than dev dependencies. For example, bootstrap, jquery, backbone, eonasdan-bootstrap-datetimepicker, underscore, moment.